src/explore-education-statistics-admin/src/pages/release/datablocks/ReleaseDataBlockEditPage.tsx

```diff
@@ -105,12 +116,28 @@ const ReleaseDataBlockEditPage = ({
       <LoadingSpinner loading={isLoading}>
         <h2>{canUpdateRelease ? 'Edit data block' : 'View data block'}</h2>
 
-        <DataBlockSelector
-          canUpdate={canUpdateRelease}
-          publicationId={publicationId}
-          releaseId={releaseId}
-          dataBlockId={dataBlockId}
-        />
+        <p className="govuk-!-margin-bottom-1">
+          {canUpdateRelease
```

Comment:
I might be wrong, but could `DataBlockSelector`'s label prop be used instead of having this separate p tag? So just have `label={canUpdateRelease}`?

Or even remove the label prop completely and just refer to canUpdate to display the text?

Reply:
I needed this with a separate tag so that the dropdown, "or" and datablock button could all be aligned horizontally and centrally in a straight line. If it has a label it adds extra height to the div.